Description: Don't install static libraries or headers.
Author: Doug Torrance <dtorrance@debian.org>
Last-Update: 2020-05-05

--- a/lib-src-reg/Makefile.am
+++ b/lib-src-reg/Makefile.am
@@ -1,11 +1,9 @@
-lib_LIBRARIES = libCHECKREG.a
+noinst_LIBRARIES = libCHECKREG.a
 
 libCHECKREG_a_SOURCES = \
 SPXinterface.cc \
 LPinterface.cc \
-RegularityCheck.cc
-
-include_HEADERS = \
+RegularityCheck.cc \
 SPXinterface.hh \
 RegularityCheck.hh \
 LPinterface.hh
--- a/lib-src/Makefile.am
+++ b/lib-src/Makefile.am
@@ -1,4 +1,4 @@
-lib_LIBRARIES       = libTOPCOM.a
+noinst_LIBRARIES       = libTOPCOM.a
 
 libTOPCOM_a_SOURCES = \
 Admissibles.cc \
@@ -34,9 +34,7 @@
 Vector.cc \
 VertexFacetTable.cc \
 VertexFacetTableArray.cc \
-VirtualChiro.cc
-
-include_HEADERS = \
+VirtualChiro.cc \
 Admissibles.hh \
 Array.hh \
 CheckTriang.hh \
--- a/wrap-gmp-gmpxx/Makefile.am
+++ b/wrap-gmp-gmpxx/Makefile.am
@@ -1,4 +1,4 @@
-include_HEADERS = \
+noinst_HEADERS = \
 Integer.h \
 Rational.h
 
