Description: Fix typo.
Author: Doug Torrance <dtorrance@debian.org>
Last-Update: 2020-05-05

--- a/lib-src/SymmetricFlipGraph.cc
+++ b/lib-src/SymmetricFlipGraph.cc
@@ -1233,7 +1233,7 @@
 	  exit(1);
 	}
 	if (CommandlineOptions::debug()) {
-	  std::cerr << "no of points in input conincides with no of points in dump file: okay" << std::endl;
+	  std::cerr << "no of points in input coincides with no of points in dump file: okay" << std::endl;
 	}
       }
       if (keyword == "_rank") {
@@ -1251,7 +1251,7 @@
 	  exit(1);
 	}
 	if (CommandlineOptions::debug()) {
-	  std::cerr << "rank of input conincides with rank of dump file: okay" << std::endl;
+	  std::cerr << "rank of input coincides with rank of dump file: okay" << std::endl;
 	}
       }
       if (_pointsptr) {
@@ -1290,7 +1290,7 @@
 	    exit(1);
 	  }
 	  if (CommandlineOptions::debug()) {
-	    std::cerr << "chirotope of input conincides with chirotope of dump file: okay" << std::endl;
+	    std::cerr << "chirotope of input coincides with chirotope of dump file: okay" << std::endl;
 	  }
 	}
       }
